### DISCUSSION TITLE
DEBUG - add debug var to display all bash lines

### DISCUSSION THREAD

**Discussion by edgd1er** [haugene/docker-transmission-openvpn#2094] [Ideas]
Would it be a good idea to add an env var called DEBUG that would have value true or false and that var would trigger a set -x for each bash ?

These two lines would be added to each bash script. debugging should easier for many of us ;)
```
DEBUG=${DEBUG:-"false"}
[[ ${DEBUG} != "false" ]] && set -x
```

**Comment by pkishino** [comment]
Sure, feel free to make a PR for this please :)

**Reply by edgd1er** [reply]
ok, I'll see what I can do, I have changed many scripts, I'm not sure all my changes are suitable to be merged in the project.
I'll keep you posted.

**Reply by edgd1er** [reply]
PR #2096
